src/topology.ts

```diff
@@ -200,14 +202,14 @@ const _runTopology = (spec: Spec, snapshot: Snapshot, dag: DAG): Response => {
         snapshot.status = hasErrors ? 'errored' : 'completed'
         snapshot.finished = new Date()
         // Emit
-        emitter.emit('data', snapshot)
         emitter.emit(hasErrors ? 'error' : 'done', snapshot)
```

Comment:
Since we're on cleanup mode, can we get rid of these events? We can do the same on wrapping the promise on a try/catch

Reply:
I'd punt on that for now.